src/app/modules/home/pipes/sentence-case.pipe.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ICoupon } from 'src/app/interface/icoupon';
+import { CouponCardComponent } from '../coupon/coupon.component';
+import { SentenceCasePipe } from './sentence-case.pipe';
+
+describe('SentenceCasePipe', () => {
+  const pipe = new SentenceCasePipe();
+
+  it('transforms "abc" to "Abc"', () => {
```

Comment:
following the style guids, the name of test case starts with 'should' or 'shouldn't'